brain/python/CMakeLists.txt

```diff
@@ -12,17 +13,32 @@ if( NOT Boost_PYTHON${USE_BOOST_PYTHON_VERSION}_LIBRARY OR
 endif()
 
 set(BRAIN_PYTHON_SOURCE_FILES
+  arrayHelpers.cpp
   brain.cpp
+  circuit.cpp
   spikeReportWriter.cpp
   spikeReportReader.cpp
-  spikes.cpp)
+  spikes.cpp
+  synapses.cpp
+  submodules.cpp
+  neuron/module.cpp
+  neuron/morphology.cpp
+)
+if(BBPTESTDATA_FOUND)
+  list(APPEND BRAIN_PYTHON_SOURCE_FILES test.cpp)
+endif()
 
 include_directories(${PYTHON_INCLUDE_DIRS})
 add_library(brain_python MODULE ${BRAIN_PYTHON_SOURCE_FILES})
```

Comment:
what's the missing part to use common_library() here? I'd like to have in CMake/common if applicable.